entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Containers should ideally be single purpose (no dev/production hybrid containers)